name.abuchen.portfolio/src/name/abuchen/portfolio/online/impl/eurostathicp-labels_nl.properties

```diff
@@ -32,5 +33,5 @@ region.SE = Zweden
 region.SI = Sloveni\u00EB
 region.SK = Slowakije
 region.TR = Turkije
-region.UK = Verenigd Koninkrijk
+region.UK = Verenigd Koningkrijk
```

Comment:
Dutch Wikipedia agrees with the previous spelling.

Reply:
Interesting - I find both...

As the file was originally created by Dutch native speakers, I will leave it as is